nixos/modules/programs/ssh.nix

```diff
@@ -203,6 +203,9 @@ in
     # generation in the sshd service.
     environment.etc."ssh/ssh_config".text =
       ''
+        ${cfg.extraConfig}
```

Comment:
```suggestion
# To allow users to override existing configuration, we prepend `extraConfig`.
${cfg.extraConfig}
```

Reply:
Is this comment aimed at people inspecting `/etc/ssh/ssh_config`, or people reading `ssh.nix`?

Reply:
Ah right. Your version makes more sense.